### DISCUSSION TITLE
Log file over 64 GB ??

### DISCUSSION THREAD

**Discussion by Cren0** [TumblThreeApp/TumblThree#269] [Q&A (Question & Answer)]
I just want to ask, as I was wondering that my HDD space has diminished a lot recently when using this app ? -so I ran a few searches, I encountered a really big file under /localdrive/appdata/local/tumblrthree/tumblrthee.log that was over 64 GB ?

Now is this something that gathers something specially important, or how can I stop this from happening, or can please someone explain....what is this for ?

**Comment by thomas694** [comment]
I've never had such a big log file. There must be tons of error messages of failed post downloads inside. A log file helps in finding a problem or errors.
Anyway, you can safely delete it. If you don't like logging, you can change the level in the settings or even switch it off.

**Comment by Cren0** [comment]
Ok thanks...it helped a lot...and switched logging off. Big Thanks !
